setup.py

```diff
@@ -51,10 +43,14 @@ def _parse_requirements(path, encoding="utf-8"):
     author="Galileo-Galilei",
     entry_points={
         "kedro.project_commands": [
-            "kedro_mlflow =  kedro_mlflow.framework.cli.cli:commands"
+            "kedro_mlflow =  kedro_mlflow.framework.cli.cli:commands",
         ],
         "kedro.global_commands": [
-            "kedro_mlflow =  kedro_mlflow.framework.cli.cli:commands"
+            "kedro_mlflow =  kedro_mlflow.framework.cli.cli:commands",
+        ],
+        "kedro.hooks": [
```

Comment:
This must be removed because of the kedro-viz bug. We can eventually modify mlflow_init to add what is necessary in the `kedro.yml` file.